src/components/ItemEditorPage/RightPanel/RightPanel.tsx

```diff
@@ -255,7 +254,7 @@ export default class RightPanel extends React.PureComponent<Props, State> {
       <div className="RightPanel">
         {isConnected ? (
           <ItemProvider id={selectedItemId}>
-            {(item, collection, isItemLoading) => {
+            {(item, _collection, isItemLoading) => {
```

Comment:
Why the undescore?

Reply:
If you mean to say that collection is an unused variable I recommend you just write an underscore without the name. just undescore means unused, undescore and name means private (in vanilla js)

Reply:
in typescript the leading underscore does tell the linter to ignore that param for the unused param rule, without it the linter would complain. Just `_` does work too, but is more vague, and you can't use it if you need to apply it on a few params but not all.